REFERENCE.md

```diff
@@ -89,6 +89,7 @@ interface AdvancedSettings {
   maxStalledCount: number = 1; // Max amount of times a stalled job will be re-processed.
   guardInterval: number = 5000; // Poll interval for delayed jobs and added jobs.s
   retryProcessDelay: number = 5000; // delay before processing next job in case of internal error.
+  backoffStrategies: {}; // A set of custom backoff strategies keyed by name. The value should be a function that when invoked, returns the delay in milliseconds.
```

Comment:
maybe an example would be useful since it may a bit difficult to understand that you have to define the strategy in the queue and then select it when you add the job to the queue.

Reply:
I added more details here and also added a full example in PATTERNS.md.